test/unix/common.ml

```diff
@@ -82,21 +82,24 @@ struct
   type t = {
     rw : Index.t;
     tbl : (string, string) Hashtbl.t;
+    cache : Index.cache;
```

Comment:
It does not matter too much for these tests (as each test use a fresh instance of the index), but before all tests used to shared the same cache. You could make the cache a global variable to simulate that behaviour. And there isn't any tests that uses the cache anyway, so no need to have it in the return type.

Reply:
> but before all tests used to shared the same cache. You could make the cache a global variable to simulate that behaviour.

I'd rather encourage the tests to be completely isolated and, as you say, it doesn't change existing semantics.

> And there isn't any tests that uses the cache anyway, so no need to have it in the return type.

Good point; removed. Not sure why I added it in the first place...